src/get-started/beam-overview.md

```diff
@@ -60,6 +61,23 @@ Beam currently supports Runners that work with the following distributed process
   <td>Apache Spark</td>
   <td>In Development</td>
 </tr>
+<tr>
+  <td>Google Cloud Dataflow</td>
+  <td>In Development</td>
+</tr>
 </table>
 
 **Note:** You can always execute your pipeline locally for testing and debugging purposes.
+
+## Getting Started with Apache Beam
+
+Interested in working with Apache Beam? Great! Here's how to get started:
+
+* If you are interested in using Beam for your data processing tasks, get started by following the [Quickstart]({{ site.baseurl }}/get-started/quickstart) and the [WordCount Examples Walkthrough]({{ site.baseurl }}/get-started/wordcount-example). Then dive into the [Documentation]({{ site.baseurl }}/documentation/) section to get up to speed on the details.
+* If you're interested in contributing to the Beam SDKs or Runners, start with the [Contribute]({{ site.baseurl }}/contribute/) section.
+
```

Comment:
the following two sections can be cut -- this should be on the landing page, no need right here